### ISSUE TITLE
Static Content-Type Bug

### ISSUE THREAD

**Issue by austinh** [socketio/socket.io#506]
When issuing static content, if you request anything other than the mime types listed (currently .js and .swf), the node server will crash and throw this error:

/usr/local/lib/node_modules/socket.io/lib/manager.js:718
      'Content-Type': mime.contentType
                          ^
TypeError: Cannot read property 'contentType' of undefined
    at serve (/usr/local/lib/node_modules/socket.io/lib/manager.js:718:27)
    at /usr/local/lib/node_modules/socket.io/lib/manager.js:746:7
    at [object Object].<anonymous> (fs.js:107:5)
    at [object Object].emit (events.js:61:17)
    at afterRead (fs.js:878:12)
    at wrapper (fs.js:245:17)

Basically, there should be a 404 error or something in there for static content if it's not one of the mime types. Also socket.io.min.js does not work (it returns empty). I think this is because it says "/socket.io.js.min" in the Manager.static dictionary, but the .min mime extension obviously doesn't work because of the bug above. It should be /socket.io.min.js
